EKF/ekf_helper.cpp

```diff
@@ -708,6 +708,31 @@ void Ekf::getFlowInnovRatio(float &flow_innov_ratio) const
 	flow_innov_ratio = _optflow_test_ratio;
 }
 
+Vector2f Ekf::getFlowVelBody() const
+{
+	return _flow_vel_body;
+}
+
+Vector2f Ekf::getFlowVelNE() const
+{
+	return _flow_vel_ne;
+}
+
+Vector2f Ekf::getFlowCompensated() const
+{
+	return _flow_compensated_XY_rad;
+}
+
+Vector2f Ekf::getFlowUncompensated() const
+{
+	return _flow_sample_delayed.flow_xy_rad;
```

Comment:
Is it necessary to expose the incoming sample?

Reply:
Well, this is the "delta angle" synchronized and integrated to the optical flow update period. It is not just a pure gyro measurement and is only an "incoming sample" if the optical flow message provides gyro data, which is often not the case.

Reply:
Actually, thanks to that logging, I found out that the optical flow scaling is off by quite a bit.